rpcs3/main_application.cpp

```diff
@@ -96,7 +97,8 @@ EmuCallbacks main_application::CreateCallbacks()
 
 	callbacks.init_pad_handler = [this](std::string_view title_id)
 	{
-		g_fxo->init<named_thread<pad_thread>>(get_thread(), m_game_window, title_id);
+		ensure(g_fxo->init<named_thread<pad_thread>>(get_thread(), m_game_window, title_id));
+		while (pad::g_reset) std::this_thread::yield();
```

Comment:
Turns out there's a rare race on master that if pad_thread::Init() is not called before cellPad it breaks, this affects savestates directly because they cellPad right away.

Reply:
Well what's the race?